torchgeo/datasets/eudem.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""European Digital Elevation Model (EU-DEM) dataset."""
```

Comment:
Does EU stand for European or European Union?

Reply:
The data is created by an Euroean Union institution, but it also covers countries that are not part of the European Union, so I suppose a little bit of both.